apps/emails-and-messages/src/modules/app-configuration/ui/instructions.tsx

```diff
@@ -0,0 +1,42 @@
+import { Link, Paper, Typography } from "@material-ui/core";
+import { actions, useAppBridge } from "@saleor/app-sdk/app-bridge";
+import { makeStyles } from "@saleor/macaw-ui";
+
+const useStyles = makeStyles((theme) => {
+  return {
+    instructionsContainer: {
+      padding: 15,
+    },
+  };
+});
+
+export const Instructions = () => {
+  const styles = useStyles();
+
+  const { appBridge } = useAppBridge();
+
+  return (
+    <Paper elevation={0} className={styles.instructionsContainer}>
+      <Typography paragraph variant="h4">
+        Usage instructions
+      </Typography>
+      <Typography paragraph>
+        Example of the external link to the documentation{" "}
+        <Link
+          href="#"
```

Comment:
href should be valid link, for a11y, it will not work anyway because of onClick (just ensure e.preventDefault is added)

Reply:
Component was copied from the template. Removed it and created task #252 for improved instructions